### PR TITLE
CrowdStrike Falcon - Add User-Agent header

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-23714

## Description
Added the header `['User-Agent'] = 'PANW-XSOAR'` to the request in order to avoid Connection issues

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by DeanArbel — APPROVED** [review]
Looks good.
@ShirleyDenkberg can you please review this?

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_17.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### CrowdStrike Falcon
+
+- Fixed an issue where ***fetch-incidents*** failed due to missing http header.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_17.md:6]:
```suggestion
- Fixed an issue where the ***fetch-incidents*** command failed due to a missing http header.
```

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_17.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- Updated the docker image to: *demisto/python3:3.10.11.56857*.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_10_17.md:7]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.11.56857*.
```

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

**Pull request merged**